apisix/config.yaml

```diff
@@ -0,0 +1,22 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+socket:
+  file: $env.APISIX_LISTEN_ADDRESS
+
+logging:
+  level: debug
```

Comment:
Is `debug` suitable as the default log level?

Reply:
I think `error` is better as the default log level, what do you think.

Reply:
`warn` would be better? Sometimes we need to tell the user something not relative to an error.

Reply:
done